### DISCUSSION TITLE
How do I use the extension? UI shows nothing.

### DISCUSSION THREAD

**Discussion by Lewdicruous** [Bing-su/adetailer#289] [Q&A]
I installed it correctly but see no way to use it. And there's no change to the generated images.
Even the tab in settings is completely blank.
![Screenshot (15)](https://github.com/Bing-su/adetailer/assets/64972683/a8e0924d-0a42-4dae-96b2-5873ba6fc90e)
![Screenshot (16)](https://github.com/Bing-su/adetailer/assets/64972683/0a21cc23-19ce-4fb4-beab-8a2f1b3f0e8b)
![Screenshot (17)](https://github.com/Bing-su/adetailer/assets/64972683/d1f791b2-2e36-429d-9bee-086ac9ed21a0)

**Comment by Bing-su** [comment]
콘솔 로그에서 관련된 에러를 보실 수 있으실겁니다.

**Reply by Lewdicruous** [reply]
There is no error shown to me.
```
venv "C:\Diffusion\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.5.1
Commit hash: 68f336bd994bed5442ad95bad6b6ad5564a5409a



Launching Web UI with arguments: --xformers --no-half
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
[-] ADetailer initialized. version: 23.8.0, num models: 10
Loading weights [6e430eb514] from C:\Diffusion\stable-diffusion-webui\models\Stable-diffusion\anything-v4.5-pruned.safetensors
Creating model from config: C:\Diffusion\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 29.2s (launcher: 10.8s, import torch: 5.8s, import gradio: 1.3s, setup paths: 1.7s, other imports: 1.6s, setup codeformer: 0.1s, list SD models: 0.4s, load scripts: 5.6s, create ui: 1.3s, gradio launch: 0.3s).
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: C:\Diffusion\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.ckpt
Applying attention optimization: xformers... done.
Model loaded in 11.2s (load weights from disk: 1.4s, create model: 0.7s, apply weights to model: 6.8s, load VAE: 0.5s, move model to device: 1.6s, calculate empty prompt: 0.2s).


```

**Reply by ljzyou** [reply]
it seems like and if you are certain the adetail extension is correctly installed,then restart web ui terminal few times to check wheher the tab changes.i had the same problem before and and restart my webui then it got fixed.And if u still dont have a clue,follow the steps in the video of this website from below:
https://www.bilibili.com/video/BV1YG411R7um/?spm_id_from=333.788.recommend_more_video.1&vd_source=328b575ee68cdb5997f2d73001106401
